File: code/e5131.pas
program Bai131;
var n, a, b, c :integer;
begin
  readln(n);
  a := n div 100;
  b := n div 10 mod 10;
  c := n mod 10;
  if (a < b) and (b < c) then writeln('YES')
  else
  writeln ('NO');
end.

File: code/e5131.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,a,b,c;
    cin>>n;
    a=n/100;
    b=n/10%10;
    c=n%10;
    if (a<b && b<c)
        cout<<"YES";
    else cout<<"NO";
    return 0;
}